Code/StudySpark.GUI.WPF/Core/FilteredBeerList.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace StudySpark.GUI.WPF.Core
+{
+    class FilteredBeerList
+    {
+        public List<string> SetFilteredList(List<bool?> filters)
+        {
+            bool HertogJanChecked = (bool) filters[0];
+            bool AmstelChecked = (bool)filters[1];
+            bool HeinekenChecked = (bool)filters[2];
+            bool GrolschChecked = (bool)filters[3];
+            bool KratChecked = (bool)filters[4];
+            bool BlikChecked = (bool)filters[5];
+            bool FlesChecked = (bool)filters[6];
+            bool FustChecked = (bool)filters[7];
+            bool TrayChecked = (bool)filters[8];
+
+            List<string> FilteredList = new();
+
+
+            if (HertogJanChecked)
+            {
+                FilteredList.Add("Hertog Jan");
+            }
+            if (AmstelChecked)
+            {
+                  FilteredList.Add("Amstel");
+            }
+            if (HeinekenChecked)
+            {
+                FilteredList.Add("Heineken");
+            }
+            if (GrolschChecked)
+            {
+                 FilteredList.Add("Grolsch");
+            }
+
+
+            if (KratChecked)
```

Comment:
Is het hier niet handiger om te zoeken naar keywords ipv. exact matches
eg. een filter zoals "\*blik\*" laten zoeken in de beschrijving
"**Blik** van 0,50 liter"
"**Blik**je van 0,33 liter"
"Set van 6 **blik**jes 0,33 liter"
"Set van 6 **blik**ken á 0,50 liter"
"Set van 8 **blik**ken á 0,50 liter"
"Set van 12 **blik**jes á 0,33 liter"

Maar dat zou betekenen dat de filter ook triggert op "Tray met 24 **blik**jes 0,33 liter". Naar mijn idee is dat niet erg.

Reply:
Oke terugkomend op de change. Dit gaat anders dan verwacht.
Wanneer je alleen flesjes selecteerd krijg je ook kratten te zien omdat "Krat van 24 **fles**jes"

Reply:
Je zou de filters ook omgekeerd kunnen toepassen. I.p.v. beschrijven moet het woord fles bevatten veranderen naar beschrijving mag niet het woord fles bevatten.